src/Error/FormattedError.php

```diff
@@ -110,7 +102,9 @@ private static function highlightSourceAtLocation(Source $source, SourceLocation
         $lineNum = (string) $contextLine;
         $nextLineNum = (string) ($contextLine + 1);
         $padLen = strlen($nextLineNum);
+
         $lines = preg_split('/\r\n|[\n\r]/', $source->body);
+        assert(is_array($lines), 'Should never be false given the regex is correct');
```

Comment:
nit: I'd expect 
```suggestion
        assert($lines !== false, 'Should never be false given the regex is correct');
```

given `Should never be false` message